tests/www/geiq_views/__snapshots__/tests.ambr

```diff
@@ -997,7 +997,7 @@
               
           </ul>
           
-              <a class="btn btn-secondary btn-block mt-4" href="/company/[PK of Company]/card">Voir la fiche de l'entreprise</a>
+              <a class="btn btn-secondary btn-block mt-4" href="/company/[PK of Company]/card?">Voir la fiche de l'entreprise</a>
```

Comment:
c'est normal que le ? apparaisse même quand il n'y a pas de job_seeker ?

Reply:
Je me disais que c'était un moindre mal. 
Les deux variables `back_url` et pour `job_seeker` peuvent être absentes ou présentes.
On a quelque chose comme ça pour le moment (par défaut on affiche toujours `?`) : 

```
<a href="{% url 'companies_views:card' siae_id=company.pk %}?{% if job_seeker|default:None %}job_seeker={{ job_seeker.public_id }}&{% endif %}{% if back_url|default:"" and not open_in_tab|default:False %}back_url={{ back_url|urlencode }}{% endif %}"
```

On pourrait afficher le `?`, si `job_seeker or back_url`. Et il faudrait aussi afficher le `&`, si `job_seeker and back_url`.
Mais ça complexifie un peu la lecture de l'URL un peu chargée déjà ?